conftest.py

```diff
@@ -90,6 +91,31 @@ def kill_all_test_spawned_processes():
     except Exception:
         print("Not all processes were killed after test")
 
+def get_hostlist():
+    global test_hostlist
+    if not test_hostlist:
+        if "COBALT_NODEFILE" in os.environ:
+            try:
+                with open(os.environ["COBALT_NODEFILE"], 'r') as nodefile:
```

Comment:
Sometimes these node files contain duplicate node names if the allocation was obtained with more than 1 rank per node. we should be sure that there aren't duplicates here as that would mess up the database deployment

Reply:
OK, I removed duplicates using the `list(dictfromkeys(...))` trick.